kanidmd_web_ui/src/components/change_unix_password.rs

```diff
@@ -0,0 +1,237 @@
+            Msg::Submit(data) => {
+                let fd: FormValues = data.into();
+                if fd.password_input != fd.password_repeat_input {
+                    return self.update(
+                        ctx,
+                        Msg::Error {
+                            emsg: "Password fields did not match".to_string(),
+                            kopid: None,
+                        },
+                    );
+                }
+                let tk = ctx.props().token.clone();
+                ctx.link().send_future(async {
+                    match Self::update_unix_password(tk, fd.password_input).await {
+                        Ok(v) => v,
+                        Err(v) => v.into(),
+                    }
+                });
+                false
+            }
+            Msg::Error { emsg, kopid } => {
+                self.state = State::Error { emsg, kopid };
+                true
+            }
+            Msg::Success => {
+                utils::modal_hide_by_id(crate::constants::ID_UNIX_PASSWORDCHANGE);
+                self.state = State::Init;
+                true
+            }
+        }
+    }
+
+    fn view(&self, ctx: &Context<Self>) -> Html {
+        let flash = match &self.state {
+            State::Error { emsg, kopid } => {
+                let message = match kopid {
+                    Some(k) => format!("An error occured - {} - {}", emsg, k),
+                    None => format!("An error occured - {} - No Operation ID", emsg),
+                };
+                html! {
+                  <div class="alert alert-danger alert-dismissible fade show" role="alert">
+                    { message }
+                    <button type="button" class="btn btn-close" data-dismiss="alert" aria-label="Close"></button>
+                  </div>
+                }
+            }
+            _ => html! { <></> },
+        };
+        html! {
+          <>
+            <button type="button" class="btn btn-primary"
+            data-bs-toggle="modal"
+            data-bs-target={format!("#{}", crate::constants::ID_UNIX_PASSWORDCHANGE)}
+            >
+              { "Update your Unix Password" }
+            </button>
+            <div class="modal" tabindex="-1" role="dialog" id={crate::constants::ID_UNIX_PASSWORDCHANGE}>
+              <div class="modal-dialog" role="document">
+                  <form
+                      onsubmit={
+                        ctx.link().callback(|e: FocusEvent| {
+                          e.prevent_default();
+                          let form = e.target().and_then(|t| t.dyn_into::<HtmlFormElement>().ok()).unwrap();
+                          Msg::Submit(FormData::new_with_form(&form).unwrap())
+                        })
+                      }
+                  >
+                    <div class="modal-content">
+                    <div class="modal-header">
+                        <h5 class="modal-title">{"Update your password"}</h5>
```

Comment:
I think it would be good here to have the title to match saying "Update your unix password" along with some short text about what this password is used for. Something like "This password is used when logging into a unix-like system"